### ISSUE TITLE
YAML files in the `.vsts-ci` directory use old syntax

### ISSUE THREAD

**Issue by DavidStaheli** [pypa/pipenv#3164]
### Issue description
YAML files in the `.vsts-ci` directory use old syntax.  For example, instead of `phases`, it should use `jobs`. Also, the `.vsts-ci` directory name uses the product's old name (before it was renamed to Azure Pipelines).

### Expected result

Updated syntax and product name.

-------------------------------------------------------------------------------

<details><summary>$ pipenv --support</summary>

Pipenv version: `'2018.10.13'`

Pipenv location: `'c:\\users\\dastahel\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\pipenv'`

Python location: `'c:\\users\\dastahel\\appdata\\local\\programs\\python\\python37\\python.exe'`

Python installations found:

  - `5.1.0`: `C:\Anaconda3-64\python.exe`
  - `3.7.0`: `C:\Users\dastahel\AppData\Local\Programs\Python\Python37\python.exe`
  - `3.6.3`: `C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python36_64\python.exe`

PEP 508 Information:

```
{'implementation_name': 'cpython',
 'implementation_version': '3.7.0',
 'os_name': 'nt',
 'platform_machine': 'AMD64',
 'platform_python_implementation': 'CPython',
 'platform_release': '10',
 'platform_system': 'Windows',
 'platform_version': '10.0.17763',
 'python_full_version': '3.7.0',
 'python_version': '3.7',
 'sys_platform': 'win32'}
```

System environment variables:

  - `ALLUSERSPROFILE`
  - `APPDATA`
  - `APR_ICONV_PATH`
  - `ASL.LOG`
  - `CHOCOLATEYINSTALL`
  - `CHOCOLATEYLASTPATHUPDATE`
  - `COMMONPROGRAMFILES`
  - `COMMONPROGRAMFILES(X86)`
  - `COMMONPROGRAMW6432`
  - `COMPUTERNAME`
  - `COMSPEC`
  - `DAVID1`
  - `DRIVERDATA`
  - `FSHARPINSTALLDIR`
  - `GOROOT`
  - `HOMEDRIVE`
  - `HOMEPATH`
  - `JAVA_HOME`
  - `JAVA_HOME_10_X64`
  - `LOCALAPPDATA`
  - `LOGONSERVER`
  - `M2_HOME`
  - `NUMBER_OF_PROCESSORS`
  - `ONEDRIVE`
  - `OS`
  - `PATH`
  - `PATHEXT`
  - `PROCESSOR_ARCHITECTURE`
  - `PROCESSOR_IDENTIFIER`
  - `PROCESSOR_LEVEL`
  - `PROCESSOR_REVISION`
  - `PROGRAMDATA`
  - `PROGRAMFILES`
  - `PROGRAMFILES(X86)`
  - `PROGRAMW6432`
  - `PROMPT`
  - `PSMODULEPATH`
  - `PUBLIC`
  - `RUBY_2_3_X64`
  - `RUBY_2_4_X64`
  - `RUBY_2_5_X64`
  - `SYSTEMDRIVE`
  - `SYSTEMROOT`
  - `TEMP`
  - `TMP`
  - `UATDATA`
  - `USERDNSDOMAIN`
  - `USERDOMAIN`
  - `USERDOMAIN_ROAMINGPROFILE`
  - `USERNAME`
  - `USERPROFILE`
  - `WINDIR`
  - `PYTHONDONTWRITEBYTECODE`
  - `PIP_SHIMS_BASE_MODULE`
  - `PIP_PYTHON_PATH`

Pipenvûspecific environment variables:

Debugûspecific environment variables:

  - `PATH`: `C:\Program Files\Docker\Docker\Resources\bin;C:\ProgramData\Oracle\Java\javapath;C:\Program Files (x86)\Common Files\Oracle\Java\javapath;c:\ruby24\bin;C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\wbin;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\dotnet\;C:\Gradle\gradle-4.4.1\bin;C:\Program Files\Java\jdk1.8.0_144\bin;C:\ProgramData\chocolatey\bin;C:\Program Files (x86)\Subversion\bin;C:\Program Files (x86)\WinMerge;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\Git\cmd;C:\apache-maven-3.5.4\bin;C:\Program Files\nodejs\;C:\Users\dastahel\AppData\Local\Programs\Python\Python37\Scripts\;C:\Users\dastahel\AppData\Local\Programs\Python\Python37\;C:\Users\dastahel\AppData\Local\Microsoft\WindowsApps;C:\Program Files\Microsoft VS Code\bin;C:\Users\dastahel\AppData\Local\Programs\Fiddler;C:\Users\dastahel\AppData\Local\Microsoft\WindowsApps;C:\Users\dastahel\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\dastahel\AppData\Roaming\npm;C:\Users\dastahel\AppData\Local\now-cli`

---------------------------

</details>

**Comment by techalchemy** [comment]
THANK YOU

I have so much trouble staying on top of CI syntax. Seriously, VSTS is amazing.  The test analytics are best in class.  I have never regretted making the switch.

**Comment by DavidStaheli** [comment]
Wow - thanks a lot, Dan!  I'm all ears if there's anything we can do to improve it for you.

**Comment by divais** [comment]
>           THANK YOU
> I have so much trouble staying on top of CI syntax. Seriously, VSTS is amazing.  The test analytics are best in class.  I have never regretted making the switch.

Thank you Dan!! Do share if you any specific feedback on test analytics.

**Comment by frostming** [comment]
Closed via #3165
